apps/web/src/components/Profile/index.tsx

```diff
@@ -27,9 +28,10 @@ const ViewProfile: NextPage = () => {
       : 'FEED'
   );
 
+  const handle = formatHandle(username as string);
   const { data, loading, error } = useProfileQuery({
-    variables: { request: { handle: username }, who: currentProfile?.id ?? null },
-    skip: !username
+    variables: { request: { handle: handle }, who: currentProfile?.id ?? null },
```

Comment:
```suggestion
    variables: { request: { handle }, who: currentProfile?.id ?? null },
```

you'd not have to de structure it anyways as handle is already the variable name?